chart/flux/templates/secret.yaml

```diff
@@ -3,5 +3,8 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: {{ template "flux.fullname" . }}-git-deploy
+  {{- if .Values.serviceAccount.annotations }}
```

Comment:
This does not seem to be the right `.Values` given the addition to the `values.yaml`.

Reply:
You are correct, I did some copy and paste and I missed this.